integrations/opsgenie/README.md

```diff
@@ -0,0 +1,387 @@
+resources:
+  - kind: alert
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .message
+          blueprint: '"opsGenieAlert"'
+          properties:
+            status: .status
+            acknowledged: .acknowledged
+            responders: .responders
+            priority: .priority
+            sourceName: .source
+            tags: .tags
+            count: .count
+            createdBy: .owner
+            createdAt: .createdAt
+            updatedAt: .updatedAt
+            description: .description
+            integration: .integration.name
+```
+</details>
+
+### Incident
+The mapping should refer to one of the incidents in the example response: [OpsGenie documentation](https://docs.opsgenie.com/docs/incident-api#list-incidents)
+
+<details>
+<summary>blueprint.json</summary>
+
+```json
+{
+	"identifier": "opsGenieIncident",
+	"description": "This blueprint represents an OpsGenie incident in our software catalog",
+	"title": "OpsGenie Incident",
+	"icon": "OpsGenie",
+	"schema": {
+		"properties": {
+			"description": {
+				"title": "Description",
+				"type": "string"
+			},
+			"status": {
+				"type": "string",
+				"title": "Status",
+				"enum": [
+					"closed",
+					"open",
+					"resolved"
+				],
+				"enumColors": {
+					"closed": "blue",
+					"open": "red",
+					"resolved": "green"
+				},
+				"description": "The status of the incident"
+			},
+			"url": {
+				"type": "string",
+				"format": "url",
+				"title": "URL"
+			},
+			"tags": {
+				"type": "array",
+				"items": {
+					"type": "string"
+				},
+				"title": "Tags"
+			},
+			"responders": {
+				"type": "array",
+				"title": "Responders",
+				"description": "Responders to the alert"
+			},
+			"priority": {
+				"type": "string",
+				"title": "Priority"
+			},
+			"createdAt": {
+				"title": "Create At",
+				"type": "string",
+				"format": "date-time"
+			},
+			"updatedAt": {
+				"title": "Updated At",
+				"type": "string",
+				"format": "date-time"
+			}
+		},
+		"required": []
+	},
+	"mirrorProperties": {},
+	"calculationProperties": {},
+	"relations": {
+		"services": {
+			"title": "Impacted Services",
+			"target": "opsGenieService",
+			"many": true,
+			"required": false
+		}
+	}
+}
+```
+</details>
+<details>
+  <summary>port-app-config.yaml</summary>
+
+```yaml
+resources:
+  - kind: incident
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .message
+          blueprint: '"opsGenieIncident"'
+          properties:
+            status: .status
+            responders: .responders
+            priority: .priority
+            tags: .tags
+            url: .links.web
+            createdAt: .createdAt
+            updatedAt: .updatedAt
+            description: .description
+          relations:
+            services: .impactedServices
+```
+</details>
+
+## Webhook Configuration
```

Comment:
Change the section to "Configuring real-time updates".

And, write the documentation about how to create a webhook here, don't forward to Port's docs as it explains about the webhook feature which does not fit here